From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mpv-winbuild CI <mpv-winbuild@localhost>
Date: Sat, 29 Aug 2026 00:00:00 +0000
Subject: [PATCH] Only polyfill wcscpy_s on pre-2005 MSVC

`#if _MSC_VER < 1400` is true when _MSC_VER is undefined (clang/mingw),
which defines wcscpy_s macros that break mingw stralign.h.
Context regenerated from the real intel/libvpl v2.17.0 file (the body
of the #if block is indented four spaces upstream).
---
 libvpl/src/windows/mfx_dispatcher_defs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libvpl/src/windows/mfx_dispatcher_defs.h b/libvpl/src/windows/mfx_dispatcher_defs.h
index 1111111..2222222 100644
--- a/libvpl/src/windows/mfx_dispatcher_defs.h
+++ b/libvpl/src/windows/mfx_dispatcher_defs.h
@@ -17,7 +17,7 @@
 #define MAX_PLUGIN_PATH 4096
 #define MAX_PLUGIN_NAME 4096
 
-#if _MSC_VER < 1400
+#if defined(_MSC_VER) && _MSC_VER < 1400
     #define wcscpy_s(to, to_size, from) \
         (void)(to_size);                \
         wcscpy(to, from)
